client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -2011,6 +2012,58 @@ public void testCreateSnapshot() throws IOException {
         assertToXContentBody(createSnapshotRequest, request.getEntity());
     }
 
+    public void testGetSnapshots() {
+        Map<String, String> expectedParams = new HashMap<>();
+        String repository = randomIndicesNames(1, 1)[0];
+        String snapshot1 = "snapshot1-" + randomAlphaOfLengthBetween(2, 5).toLowerCase(Locale.ROOT);
+        String snapshot2 = "snapshot2-" + randomAlphaOfLengthBetween(2, 5).toLowerCase(Locale.ROOT);
+
+        String endpoint = String.format(Locale.ROOT, "/_snapshot/%s/%s,%s", repository, snapshot1, snapshot2);
+
+        GetSnapshotsRequest getSnapshotsRequest = new GetSnapshotsRequest();
+        getSnapshotsRequest.repository(repository);
+        getSnapshotsRequest.snapshots(Arrays.asList(snapshot1, snapshot2).toArray(new String[0]));
+        setRandomMasterTimeout(getSnapshotsRequest, expectedParams);
+        if (randomBoolean()) {
+            getSnapshotsRequest.ignoreUnavailable(true);
+            expectedParams.put("ignore_unavailable", Boolean.TRUE.toString());
+        }
+        if (randomBoolean()) {
+            getSnapshotsRequest.verbose(false);
+            expectedParams.put("verbose", Boolean.FALSE.toString());
+        }
```

Comment:
maybe just simplify these by setting it to the value of the `randomBoolean()` and using `Boolean.valueOf`? Not super necessary, just a nit.